README.md

```diff
@@ -0,0 +1,93 @@
+# Twitter Driver
+
+An [Membrane.io](https://membrane.io/) driver for Twitter.
+
+## Usage
+
+1. Install the [Membrane VSCode Extension](https://marketplace.visualstudio.com/items?itemName=membrane.membrane).
+2. Setup on extension your [Membrane's CLI binary (mctl)](https://membrane.io/download) path.
+3. Login / Sign up with ```mctl login```.
+4. Update the program on Membrane with the VSCode Command palette `(cmd+shift+p)`\
+  ```> Membrame: Update current program```
```

Comment:
We can remove this entire part as it's going to be outdated pretty soon and it's understood that you need to be using Membrane to use this driver